eng/pipelines/coreclr/perf.yml

```diff
@@ -80,199 +80,238 @@ jobs:
       buildConfig: release
       platforms:
       - Linux_x64
-      - windows_x64
-      - windows_x86
-      - Linux_arm64
-      - Linux_musl_x64
-      - windows_arm64
+      #- windows_x64
+      #- windows_x86
+      #- Linux_arm64
+      #- Linux_musl_x64
+      #- windows_arm64
       jobParameters:
         testGroup: perf
 
   # build mono on wasm
+  # - template: /eng/pipelines/common/platform-matrix.yml
+  #   parameters:
+  #     jobTemplate: /eng/pipelines/common/global-build-job.yml
```

Comment:
These and other commented code - again assumption is it for testing only.

Reply:
Yes